docker/docker-compose.yaml

```diff
@@ -100,7 +100,7 @@ services:
       - vc_auth
 
   aca-py:
-    image: ghcr.io/hyperledger/aries-cloudagent-python:py3.9-0.12.1
+    image: ghcr.io/hyperledger/aries-cloudagent-python:py3.12-1.0.0rc4
```

Comment:
We can update this to rc5 now I believe https://github.com/hyperledger/aries-cloudagent-python/pkgs/container/aries-cloudagent-python/247708442?tag=py3.12-1.0.0rc5